iceoryx_posh/include/iceoryx_posh/runtime/posh_runtime.hpp

```diff
@@ -51,6 +51,11 @@ enum class FindServiceError
     INSTANCE_CONTAINER_OVERFLOW
 };
 
+/// @brief Used to search for any string (wildcard)
+struct Any_t
```

Comment:
We should be more explicit and call it `AnyIdString`. Although it is no string, it is used in for the string wildcard search only (e.g. has nothing to do with a general `any` wrapper which can store any object).

Reply:
You're right there. However, I'd rather change it back to `Wildcard`. `AnyIdString` is too cryptic for me personally.